{"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1993", "repository_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT", "labels_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1993/labels{/name}", "comments_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1993/comments", "events_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1993/events", "html_url": "https://github.com/Significant-Gravitas/Auto-GPT/issues/1993", "id": 1670109652, "node_id": "I_kwDOJKSTjM5ji9nU", "number": 1993, "title": "Error: Invalid JSON, setting it to empty JSON now.", "user": {"login": "petersevera", "id": 98494558, "node_id": "U_kgDOBd7oXg", "avatar_url": "https://avatars.githubusercontent.com/u/98494558?v=4", "gravatar_id": "", "url": "https://api.github.com/users/petersevera", "html_url": "https://github.com/petersevera", "followers_url": "https://api.github.com/users/petersevera/followers", "following_url": "https://api.github.com/users/petersevera/following{/other_user}", "gists_url": "https://api.github.com/users/petersevera/gists{/gist_id}", "starred_url": "https://api.github.com/users/petersevera/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/petersevera/subscriptions", "organizations_url": "https://api.github.com/users/petersevera/orgs", "repos_url": "https://api.github.com/users/petersevera/repos", "events_url": "https://api.github.com/users/petersevera/events{/privacy}", "received_events_url": "https://api.github.com/users/petersevera/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "open", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 10, "created_at": "2023-04-16T19:43:12Z", "updated_at": "2023-04-17T05:03:26Z", "closed_at": null, "author_association": "NONE", "active_lock_reason": null, "body": "### \u26a0\ufe0f Search for existing issues first \u26a0\ufe0f\n\n- [X] I have searched the existing issues, and there is no existing issue for my problem\n\n### GPT-3 or GPT-4\n\n- [ ] I am using Auto-GPT with GPT-3 (GPT-3.5)\n\n### Steps to reproduce \ud83d\udd79\n\nTHOUGHTS:  None\r\nREASONING:  None\r\nCRITICISM:  None\r\nSPEAK:  None\r\nAttempting to fix JSON by finding outermost brackets \r\nError: Invalid JSON, setting it to empty JSON now.\r\n  \r\nNEXT ACTION:  COMMAND = Error: ARGUMENTS = 'dict' object has no attribute 'replace'\r\nSYSTEM:  Command Error: threw the following error: 'dict' object has no attribute 'replace'\r\n\n\n### Current behavior \ud83d\ude2f\n\n_No response_\n\n### Expected behavior \ud83e\udd14\n\n_No response_\n\n### Your prompt \ud83d\udcdd\n\n```yaml\r\n# Paste your prompt here\r\n```\r\n", "reactions": {"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1993/reactions", "total_count": 2, "+1": 2, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1993/timeline", "performed_via_github_app": null, "state_reason": null}